<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connexionmainmenu</class>
 <widget class="QMainWindow" name="connexionmainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="MainMenustackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="formLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>410</y>
        <width>611</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="connexionformLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="nomUtilisateurlabel">
         <property name="text">
          <string>Nom d'utilisateur</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nomUtilisateurlineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="motPasselabel">
         <property name="text">
          <string>Mot de passe</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="motPasselineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="logolabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>524</width>
        <height>227</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ressources.qrc">:/img/logo_epaf.jpg</pixmap>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>470</y>
        <width>611</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="connexionButtonhorizontalLayout">
       <item>
        <widget class="QPushButton" name="connexionpushButton">
         <property name="text">
          <string>Connexion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="annulerpushButton">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>logolabel</zorder>
     <zorder>formLayoutWidget_7</zorder>
     <zorder>logolabel</zorder>
     <zorder>pushButton</zorder>
     <zorder>horizontalLayoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="MenuP" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>951</width>
        <height>481</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Menu Principal</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>821</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="Menugrid">
        <item row="0" column="0">
         <widget class="QPushButton" name="ajouterPatpushButton">
          <property name="text">
           <string>Ajouter patineur</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ajouterCoachpushButton">
          <property name="text">
           <string>Ajouter coach</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="accesRapidepushButton">
          <property name="text">
           <string>Accès rapide</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="recherchepushButton">
          <property name="text">
           <string>Recherche</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="creationRecuspushButton">
          <property name="text">
           <string>Création de reçu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>nomUtilisateurlineEdit</tabstop>
  <tabstop>motPasselineEdit</tabstop>
  <tabstop>connexionpushButton</tabstop>
  <tabstop>annulerpushButton</tabstop>
  <tabstop>accesRapidepushButton</tabstop>
  <tabstop>recherchepushButton</tabstop>
  <tabstop>creationRecuspushButton</tabstop>
  <tabstop>ajouterPatpushButton</tabstop>
  <tabstop>ajouterCoachpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>annulerpushButton</sender>
   <signal>clicked()</signal>
   <receiver>motPasselineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>annulerpushButton</sender>
   <signal>clicked()</signal>
   <receiver>nomUtilisateurlineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
